clippy_lints/src/utils/internal_lints.rs

```diff
@@ -165,10 +188,34 @@ pub struct LintWithoutLintPass {
 
 impl_lint_pass!(LintWithoutLintPass => [LINT_WITHOUT_LINT_PASS]);
 
+declare_lint_pass!(DefaultLint => [DEFAULT_LINT]);
```

Comment:
```suggestion
impl_lint_pass!(LintWithoutLintPass => [DEFAULT_LINT, LINT_WITHOUT_LINT_PASS]);
```

You have to put this in the `impl_lint_pass!` macro above

Reply:
done

Reply:
done